test/integration/package_int_test.go

```diff
@@ -755,6 +755,37 @@ func (suite *PackageIntegrationTestSuite) TestCVE_Indirect() {
 	cp.ExpectExitCode(1)
 }
 
+func (suite *PackageIntegrationTestSuite) TestChangeSummary() {
+	suite.OnlyRunForTags(tagsuite.Package)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.Spawn("config", "set", constants.AsyncRuntimeConfig, "true")
+	cp.Expect("Successfully set")
+	cp.ExpectExitCode(0)
+
+	cp = ts.SpawnWithOpts(
+		e2e.OptArgs("checkout", "ActiveState-CLI/small-python", "."),
+		e2e.OptAppendEnv(constants.DisableRuntime+"=false"),
```

Comment:
Async runtimes

Reply:
For whatever reason, if I remove this, and there is no runtime checked out, the subsequent `state install` does not show dependency and CVE information. It just immediately prints "Added package: <name>". So this has to stay, unfortunately.